parcels/examples/tutorial_parcels_structure.ipynb

```diff
@@ -0,0 +1,369 @@
+    "\n",
+    "If you want to store the particle data generated in the simulation, you usually first want to define the [**`ParticleFile`**](https://oceanparcels.org/gh-pages/html/#parcels.particlefile.ParticleFile) to which the output of the kernel execution will be written. Then, on the [`ParticleSet`](#ParticleSet) you have defined, you can use the method [**`ParticleSet.execute()`**](https://oceanparcels.org/gh-pages/html/#parcels.particleset.ParticleSet.execute) which requires the following arguments:\n",
+    "1. The kernels to be executed\n",
+    "2. The `runtime` defining how long the execution loop runs. Alternatively, you may define the `endtime` at which the execution loop stops.\n",
+    "3. The timestep `dt` at which to execute the kernels\n",
+    "4. (Optional) The `ParticleFile` object to write the output to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO: Compiled JITParticleAdvectionRK4 ==> C:\\Users\\GEBRUI~1\\AppData\\Local\\Temp\\parcels-tmp\\d8d64a9f5d2109ee91c0cae898d78aec_0.dll\n"
+     ]
+    }
+   ],
+   "source": [
+    "output_file = pset.ParticleFile(name=\"GCParticles.nc\", outputdt=3600) # the file name and the time step of the outputs\n",
+    "\n",
+    "pset.execute(AdvectionRK4,                 # the kernel (which defines how particles move)\n",
+    "             runtime=86400*6,              # the total length of the run in seconds\n",
+    "             dt=300,                       # the timestep of the kernel\n",
+    "             output_file=output_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One of the most powerful features of Parcels is the ability to write custom Kernels (see e.g. [this example](http://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/parcels/examples/parcels_tutorial.ipynb#Adding-a-custom-behaviour-kernel) in the parcels tutorial). You probably want to define these kernels in this section; after defining your [`ParticleSet`](#ParticleSet) and before executing them. If your kernels become very large or complex, you might want to store them in another file and import them into your simulation script."
```

Comment:
If we move away from the old parcels tutorial, then better to put custom kernels in a separate tutorial?

Reply:
Agreed, until then I can maybe change this link to the sampling or the Argofloats tutorial?